src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -416,29 +512,93 @@ private ProjectCalendar getResourceCalendar(Integer calendarID)
    }
 
    /**
+    * Process resource rates
+    *
+    * @param rows resource rate data
+    */
+   public void processResourceRates(List<Row> rows)
+   {
+      // Primavera defines resource cost tables by start dates so sort and define end by next
+      rows.sort(new Comparator<Row>()
```

Comment:
I'm missing a `sort` method on `List<T>`.. perhaps it's a target JDK version (we need to aim for 1.6). Try this:
```java
      Collections.sort(rows, new Comparator<Row>()
      {
```